bucket/amazon-workspaces.json

```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://clients.amazonworkspaces.com/",
+    "description": "Amazon workspaces client.",
+    "license": "Proprietary",
+    "version": "3.1.4",
+    "hash": "604e40ed03883e1d740367fbcc4da9c21fc88fe26f9df7aca4ed0a25c0b1ac2e",
+    "url": "https://d2td7dqidlhjx7.cloudfront.net/prod/global/windows/Amazon+WorkSpaces.msi",
+    "extract_dir": "[ApplicationFolderName]",
+    "bin": "workspaces.exe",
+    "shortcuts": [
+        [
+            "workspaces.exe",
+            "Amazon Workspaces"
+        ]
+    ],
+    "checkver": {
+        "url": "https://docs.aws.amazon.com/workspaces/latest/userguide/amazon-workspaces-windows-client.html",
+        "regex": "\\<td\\>(\\d+\\.\\d+\\.\\d+)\\</td\\>"
+    },
+    "autoupdate": {
+        "url": "https://d2td7dqidlhjx7.cloudfront.net/prod/global/windows/Amazon+WorkSpaces.msi"
```

Comment:
Find proper versioned URL

Reply:
There is no such thing. Amazon keeps only version 2.5.11 and 3.1.4 available. 

2.5.11 is on url https://d2td7dqidlhjx7.cloudfront.net/versions/2.5.11/windows/Amazon+WorkSpaces.msi
3.1.4 is on url https://d2td7dqidlhjx7.cloudfront.net/prod/global/windows/Amazon+WorkSpaces.msi

I tried other versions but nothing resolves